string color_image_path
---
sensor_msgs/Image grayscale_image